src/redux/sagas/appointmentSaga.ts

```diff
@@ -0,0 +1,92 @@
+import { Appointment } from "@prisma/client";
+import {
+  all,
+  call,
+  CallEffect,
+  put,
+  PutEffect,
+  takeEvery,
+} from "redux-saga/effects";
+import { IAppointment } from "src/types/appointment";
+import { IAppointmentActionSetAppointmentStatus } from "../actions/appointmentAction";
+import AppointmentTypes from "../types/appointmentTypes";
+
+function patchAcceptAppointment(
```

Comment:
```suggestion
function patchAppointmentStatus(
```

Rename since this applies updates for any status type